components/Food/FoodList.js

```diff
@@ -1,13 +1,13 @@
-import { dataFoods } from "../../lib/data";
 import styled from "styled-components";
 import { Fragment } from "react";
+import { dataFoods } from "../../lib/data";
 import FoodCard from "./FoodCard";
 
 export default function FoodList() {
   return (
     <>
-      <StyledListHeader>Obst & Gemüse</StyledListHeader>
-      <StyledList>
+      <StyledFoodListHeader>Obst & Gemüse</StyledFoodListHeader>
+      <StyledFoodList>
         {dataFoods.map((dataFood) => {
           return (
             <Fragment key={dataFood.id}>
```

Comment:
not sure if you need the Fragment here or rather a div or something to store the key. I thought Fragments are only for several elements on the same level and you only have the FoodCard in there.

Reply:
Solved together via zoom. Fragment helps for avoiding div salad.